rules/sat/sat-inscript2.js

```diff
@@ -3,103 +3,60 @@
 
 	var satInScript2 = {
 		id: 'sat-inscript2',
-		name: 'संताली इनस्क्रिप्ट २',
+		name: 'inskript2',
```

Comment:
Well I have discussed with some santali language people they have told me that this will be right. inskript2, with a small letter in the start.